### DISCUSSION TITLE
DBus dependency

### DISCUSSION THREAD

**Discussion by qarkai** [FreeRDP/FreeRDP#7655] [Q&A] [answered]
There is requirement for dbus and dbus-glib packages in https://github.com/FreeRDP/FreeRDP/wiki/Compilation#install-the-suggested-base-dependencies.
But it looks like project doesn't use DBus. I couldn't find any related code except FindDBus and FindDbusGlib cmake modules which seems to be unused.
Does this dependency still valid or it could be safely removed from deb and rpm build specs and wiki instruction?

**Comment by akallabeth** [comment, marked as answer]
The [MS-RDPEV](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpev/ff2a9f63-cbcc-4615-849f-03752a2b440b) (`-DCHANNEL_TSMF=ON`) requires `gstreamer` and all other glib dependencies.
We´ve deactivated this channel by default as a better working replacement [MS-RDPEVOR](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpevor/a9947d55-9408-4cf8-b113-555b436bd3ce) is available.

There are quite a lot of conditional dependencies, so I´ve started documenting these here: https://github.com/FreeRDP/FreeRDP/wiki/Compilation (still incomplete)
